doc/readthedocs-pip-requirements.txt

```diff
@@ -0,0 +1,13 @@
+numpydoc==0.4
+numpy
+ipython
+matplotlib
+scipy
+sphinx>=1.4
+ipykernel
+nbsphinx
+numba
+sphinx_rtd_theme
+recommonmark
+sphinx-markdown-tables
+notedown
```

Comment:
I need the last three more dependencies than clifford.

Reply:
These all seem reasonable